soc/xtensa/intel_adsp/common/include/adsp-clk.h

```diff
@@ -7,14 +7,15 @@
 #define ZEPHYR_SOC_INTEL_ADSP_CAVS_CLK_H_
```

Comment:
if you refactor anyway, can you change the header name to contain lower dash separator?
I mean "adsp_clk.h"? I think it is common style for all headers now but some old remained with dash separator.

Reply:
Done :)